docs/guidelines/content-guidelines/05-tabs.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Tabs & toggles
+---
+
+Tabs and toggles allow you to render several versions of a given section in one document or have several versions of one document on `http://kyma-project.io`.
+
+You can use tabs and/or documentation toggle to include instructions for different operating systems (OS) in one guide or to describe installation guides that differ only in a few steps.
+
+## Docsify tabs
+
+To display tabbed content from Markdown, kyma-project.io uses [docsify-tabs plugin](https://jhildenbiddle.github.io/docsify-tabs/#/) from [docsify.js](https://docsify.js.org/#/).
+
+You define a tab set using `tabs:start` and `tabs:end` HTML comments. For detailed usage guidelines see the [Usage section in the plugin official documentation](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
+
+> **CAUTION:** The use of HTML comments prevents tab-related markup from being displayed when Markdown is rendered as HTML outside of `http://kyma-project.io`, for example, in GitHub. As a result in GitHub the content displays in sections under headers defined in the tab set.
+
+## Documentation toggle
+
+The toggle renders the text in Github documents in the form of expandable sections and shows it as tabs on the `http://kyma-project.io` website.
+
+Content from documentation toggles is displayed as tabs on `http://kyma-project.io` using a dedicated script.
+
+### Writing rules
+
+Follow these rules when inserting toggles in documents:
+
+* Toggles belonging to one set must start with the `<div tabs name="{toggle-name}">` tag and end with the `</div>` tag, where **name** is a distinctive ID of a specific toggle used for linking. It can be any name describing the content of the documentation toggle.
+
+See the example:
```

Comment:
as all of this is part of a bullet list, the examples should be indented with (probably) two blank spaces so that they're rendered below the respective list item.